lambda_code/src/models/constants.py

```diff
@@ -1,12 +1,36 @@
+"""Constants"""
+
 import re
 from datetime import datetime
+from typing import Union
 
 
 class Constants:
-    genders = {"male", "female", "other", "unknown"}
-    action_flags = {"completed", "entered-in-error"}
-    vaccination_not_given_flag: str = "not-done"
-    vaccination_given_flag: str = "empty"
+    """ "Constants used for the models"""
+
+    # Constants
+    STATUSES = ("completed", "entered-in-error", "not-done")
+    PRIMARY_SOURCE = {True, False}
+    GENDERS = ("male", "female", "other", "unknown")
+    VACCINATION_NOT_GIVEN_FLAG: str = "not-done"
+    VACCINATION_GIVEN_FLAG: str = "empty"
```

Comment:
To remove, as discussed in dev stand up.

Reply:
Separate bug ticket raised